docs/getting-started.md

```diff
@@ -80,99 +84,285 @@ await Kilt.init({ address: YOUR_CHAIN_ADDRESS })
-// claimer.address: 4rjPNrzFDMrp9BudjmAV8ED7vzFBaF1Dgf8FwUjmWbso4Eyd
+const claimerMnemonic = mnemonicGenerate()
+
+const claimer = keyring.createFromUri(claimerMnemonic)
+
 console.log('claimer address', claimer.address)
 ```
 
 At this point the generated Identity has no tokens. If you want to interact with the blockchain, you will have to get some by [requesting them from our faucet](https://faucet.kilt.io/).
 
-## 3. How to build and store a Claim Type (CTYPE)
+### 2.1. Generate a Keystore
 
-When building a CTYPE, you only need a JSON schema and your public [SS58 address](<https://github.com/paritytech/substrate/wiki/External-Address-Format-(SS58)>) which you automatically receive when generating an identity.
+To create a light DID, there needs to be a keystore instance that conforms to the [Keystore interface](../types/src/Keystore.ts). For the sake of ease of use, this package includes a [demo keystore](./src/DemoKeystore/DemoKeystore.ts) which can be used to generate key pairs that are kept in memory and disappear at the end of the program execution.
 
-### 3.1. Building a CTYPE
+**Using the demo keystore in production is highly discouraged as all the keys are kept in the memory and easily retrievable by malicious actors.**
 
 ```typescript
 import Kilt from '@kiltprotocol/sdk-js'
 
-const ctype = Kilt.CType.fromSchema({
-  $schema: 'http://kilt-protocol.org/draft-01/ctype#',
-  title: 'Drivers License',
-  properties: {
-    name: {
-      type: 'string',
-    },
-    age: {
-      type: 'integer',
-    },
-  },
-  type: 'object',
+const keystore = new Kilt.Did.DemoKeystore()
+
+// Signing keypair
+const claimerSigningKeypair = await keystore.generateKeypair({
+  alg: Kilt.Did.SigningAlgorithms.Ed25519,
+  seed: generateClaimerMnemonic,
+})
+
+// Encryption keypair
+const claimerEncryptionKeypair = await keystore.generateKeypair({
+  alg: Kilt.Did.EncryptionAlgorithms.NaclBox,
+  seed: generateClaimerMnemonic,
 })
 ```
 
-### 3.2. Storing a CTYPE
+### 2.2. Generate a light DID for the Claimer
 
-Before you can store the CTYPE on the blockchain, you have to configure your blockchain address and connect to it.
-Either explicitly:
+Using the keys from the demo keystore to generate the claimers light DID.
 
 ```typescript
-Kilt.config({ address: YOUR_CHAIN_ADDRESS })
-await Kilt.connect()
+import Kilt from '@kiltprotocol/sdk-js'
+
+const claimerLightDid = new Kilt.Did.LightDidDetails({
+  authenticationKey: {
+    publicKey: claimerSigningKeypair.publicKey,
+    type: Kilt.Did.DemoKeystore.getKeypairTypeForAlg(claimerSigningKeypair.alg),
+  },
+  encryptionKey: {
+    publicKey: claimerEncryptionKeypair.publicKey,
+    type: Kilt.Did.DemoKeystore.getKeypairTypeForAlg(
+      claimerEncryptionKeypair.alg
+    ),
+  },
+})
+
+// DID light: `did:kilt:light:014qFxmHnWw5sGMwjskdvMCrASF9Jvu5ggWRTWTK2NNYSLDg56:oWFlomlwdWJsaWNLZXlYIJuIow7rjSdf92qMKYtWV42lF9mctD1nFf8RM24auJhwZHR5cGVmeDI1NTE5`
```

Comment:
This won't probably match the output DID as we use a random mnemonic for the claimer.

Reply:
Sure, I will put `Example light DID:`.